test/case/03_test_compatibility

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -x
+
+HERE=${0%/*}
+
+REPO_ROOT=$(git rev-parse --show-toplevel)
+
+source $REPO_ROOT/test/lib.sh
+
+repo=$(empty_repo)
+add_cleanup $repo
+
+bootstrap_repo $repo
+
+cd $repo
+
+# TODO: test REPO_NAME detection by setting an appropriately-formatted
+# git remote
+export REPO_NAME=${0##*/}
+
+override_boilerplate_version $(new_boilerplate_repo)/boilerplate
+
+make update_boilerplate
+
+check_update 
+
+if [ $? -ne 0 ] ; then
+	exit $?
+fi
+
+add_convention test/test_base_convention
+make update_boilerplate
+
+check_update
+
```

Comment:
Did you mean to `reset_boilerplate_version`, `make update_boilerplate`, and `check_update` again here? Else all we've done is checked `master`, which we know already passed or it wouldn't have gotten merged :P

Reply:
yes, needs to finish writing the test case